2020-10-29 17:33:31,045-INFO-my_log.py-2020_api_auto-日志信息:-----------------开始测试-----------------
2020-10-29 17:33:31,046-INFO-my_log.py-2020_api_auto-日志信息:执行用例1注册_获取图片滑动随机值
2020-10-29 17:33:31,318-INFO-my_log.py-2020_api_auto-日志信息:接收到的测试数据{}
2020-10-29 17:33:31,387-ERROR-my_log.py-2020_api_auto-日志信息:测试获取到的结果是：{'errcode': 0, 'data': {'random': '0.4949079646892208', 'verificationId': '97aed0ef-4419-4c04-a923-c634d2d1b4da'}, 'errmsg': 'ok'}
2020-10-29 17:33:34,389-INFO-my_log.py-2020_api_auto-日志信息:-----------------测试结束-------------------
2020-10-29 17:33:34,390-INFO-my_log.py-2020_api_auto-日志信息:-----------------开始测试-----------------
2020-10-29 17:33:34,391-INFO-my_log.py-2020_api_auto-日志信息:执行用例2注册_获取验证码
2020-10-29 17:34:53,600-INFO-my_log.py-2020_api_auto-日志信息:-----------------测试结束-------------------
2020-10-29 17:34:53,621-INFO-my_log.py-2020_api_auto-日志信息:-----------------开始测试-----------------
2020-10-29 17:34:53,622-INFO-my_log.py-2020_api_auto-日志信息:执行用例3注册成功
2020-10-29 17:34:53,659-INFO-my_log.py-2020_api_auto-日志信息:-----------------测试结束-------------------
2020-10-29 17:36:03,360-INFO-my_log.py-2020_api_auto-日志信息:-----------------开始测试-----------------
2020-10-29 17:36:03,361-INFO-my_log.py-2020_api_auto-日志信息:执行用例1注册_获取图片滑动随机值
2020-10-29 17:36:03,654-INFO-my_log.py-2020_api_auto-日志信息:接收到的测试数据{}
2020-10-29 17:36:03,701-ERROR-my_log.py-2020_api_auto-日志信息:测试获取到的结果是：{'errcode': 0, 'data': {'random': '0.5410485600456855', 'verificationId': 'fb6b1785-af41-4037-b50c-a7e7c76ace22'}, 'errmsg': 'ok'}
2020-10-29 17:36:06,701-INFO-my_log.py-2020_api_auto-日志信息:-----------------测试结束-------------------
2020-10-29 17:36:06,702-INFO-my_log.py-2020_api_auto-日志信息:-----------------开始测试-----------------
2020-10-29 17:36:06,702-INFO-my_log.py-2020_api_auto-日志信息:执行用例2注册_获取验证码
2020-10-29 17:37:13,155-INFO-my_log.py-2020_api_auto-日志信息:-----------------开始测试-----------------
2020-10-29 17:37:13,156-INFO-my_log.py-2020_api_auto-日志信息:执行用例1注册_获取图片滑动随机值
2020-10-29 17:37:13,475-INFO-my_log.py-2020_api_auto-日志信息:接收到的测试数据{}
2020-10-29 17:37:13,523-ERROR-my_log.py-2020_api_auto-日志信息:测试获取到的结果是：{'errcode': 0, 'data': {'random': '0.2681962442799717', 'verificationId': '81abf15c-67d6-4d82-bf56-d9a96d65609f'}, 'errmsg': 'ok'}
2020-10-29 17:37:16,524-INFO-my_log.py-2020_api_auto-日志信息:-----------------测试结束-------------------
2020-10-29 17:37:16,525-INFO-my_log.py-2020_api_auto-日志信息:-----------------开始测试-----------------
2020-10-29 17:37:16,525-INFO-my_log.py-2020_api_auto-日志信息:执行用例2注册_获取验证码
2020-10-29 17:37:17,005-INFO-my_log.py-2020_api_auto-日志信息:接收到的测试数据{"locationX": 83.0, "mobile": "15012345678", "email": "", "source": "2", "type": "1", "verificationType": "4", "verificationId": "81abf15c-67d6-4d82-bf56-d9a96d65609f"}
2020-10-29 17:37:17,052-ERROR-my_log.py-2020_api_auto-日志信息:测试获取到的结果是：{'errcode': 0, 'data': True, 'errmsg': 'ok'}
2020-10-29 17:37:20,052-INFO-my_log.py-2020_api_auto-日志信息:-----------------测试结束-------------------
2020-10-29 17:37:20,053-INFO-my_log.py-2020_api_auto-日志信息:-----------------开始测试-----------------
2020-10-29 17:37:20,054-INFO-my_log.py-2020_api_auto-日志信息:执行用例3注册成功
2020-10-29 17:37:20,393-INFO-my_log.py-2020_api_auto-日志信息:接收到的测试数据{"mobile": "15012345678", "email": "", "verifyCode": "673833", "password": "Aa123456", "checkPassword": "Aa123456"}
2020-10-29 17:37:20,439-ERROR-my_log.py-2020_api_auto-日志信息:测试获取到的结果是：{'errcode': 0, 'data': {'userAccount': '15012345678', 'token': 'eyJhbGciOiJSUzI1NiJ9.eyJleHAiOjE2MDM5NzE0MzksInN1YiI6IuS5kOWMu-S6keebmOezu-e7n-W5s-WPsCIsImFwcElkIjoiMiIsInVzZXJJZCI6NDk2NiwidXNlck5hbWUiOiIiLCJuaWNrTmFtZSI6IjE1MDEyMzQ1Njc4IiwidXNlckNvZGUiOiJUT0tFTl9UUkFZX0NVU1RPTUVSX1dFQl9jMWUzZGI5M2FmNTE0MDQ0ODI2ZDcxZjZiODMzNTFiOCJ9.CRukcKdJu8e5sPP0ntAv_WZl6WpzpD0g5v2wmEkZ-Tx8rbzw9NYdcY5XT6GhZFwgR7YOXxwUro0VAHJFotDIyiWDqbxIptDDkNX6F0eNkdrwJ8m1hMZ9vJSMten_kxRkv-yADzQVvDqFci2m42eRPLlAAJjCEUGu7nWqpWnu1E0', 'skipPageStatus': '1'}, 'errmsg': 'ok'}
2020-10-29 17:37:23,439-INFO-my_log.py-2020_api_auto-日志信息:-----------------测试结束-------------------
